AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -629,6 +636,11 @@ - (void)layoutSubviews
 
 - (void)scrollViewWillEndDragging:(UIScrollView *)scrollView withVelocity:(CGPoint)velocity targetContentOffset:(inout CGPoint *)targetContentOffset
 {
+  _deceleratingVelocity = CGPointMake(
+    scrollView.contentOffset.x - targetContentOffset->x,
```

Comment:
we can probably trust the pointer is not nil, but should we check before dereferencing it?